docs/src/api/websocket/_blockSubscribe.mdx

```diff
@@ -26,7 +26,7 @@ with the `--rpc-pubsub-enable-block-subscription` flag.
 
 ### Parameters:
 
-<Parameter name="filter" type={"string | object"} optional={true}>
+<Parameter name="filter" type={"string | object"} required={true}>
```

Comment:
Looks correct per the Rust code (ie not an `Option<...>`)
https://github.com/solana-labs/solana/blob/21b2fce6f47899eb3f8d6be1f18431c808349811/rpc/src/rpc_pubsub.rs#L536

Reply:
Weird, that means I'll have to update the JS side then 😅